manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Wii/PC%20Applications/Easy%20CSM/ECSM%201.4.exe
+  Architecture: x86
+  InstallerType: inno

```

```suggestion
  InstallerType: inno
  Scope: machine
  ProductCode: '{C8AB2AA8-501D-4553-B096-CD78EFB90366}_is1'
```
For better package matching